scene/gui/popup.cpp

```diff
@@ -182,8 +182,8 @@ void Popup::_popup(const Rect2 &p_bounds, const bool p_centered) {
 		} else {
 			set_position(p_bounds.position);
 		}
-	}
 	_fix_size();
```

Comment:
Indentation issue.

Reply:
It was a bit late, yesterday... Fixed this now.